### DISCUSSION TITLE
SMTP Connection issues with specific customer

### DISCUSSION THREAD

**Discussion by tb-bb** [postalserver/postal#3127] [Help with using Postal]
We use Postal to send (bulk) emails via our SaaS tool. The recipients of the emails are usually our customers' employees. We don't have any problems with any other customers, but we do with one. With this one customer, we can send individual emails without any problems. However, as soon as there are more, the first, sometimes the second and third emails are still delivered. But all others are no longer delivered (~5 tests with 30 recipients). We receive the following errors:

First: 421 #4.x.2 Too many messages for this session
Then: SMTPClient::Endpoint::SMTPSessionNotStartedError

Our customer has already tried various adjustments in his systems without success (unfortunately we do not know exactly what they have done). Now they have asked us whether a new SMTP connection is opened each time a single message is sent and whether all messages are sent in one SMTP connection when sending in bulk.

Can someone answer the question about the SMTP connection? Does anyone have any other general idea what the problem might be with our customer? Could it be a problem on our / Postal side?

Thanks in advance and have a great weekend

**Comment by willpower232** [comment]
Have you seen #2917? It would be good to see the worker logs for an affected message to reveal what was happening around then, you should be able to look for the alias of the message in there.
